hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/debug/DBScanner.java

```diff
@@ -114,9 +131,18 @@ public class DBScanner implements Callable<Void>, SubcommandWithParent {
       showDefaultValue = CommandLine.Help.Visibility.ALWAYS)
   private boolean showCount;
 
-  private String keySeparatorSchemaV3 =
+  @CommandLine.Option(names = {"--compact"},
+      description = "disable the pretty print the output",
+      defaultValue = "false")
+  private static boolean compact;
+
+  private static final String KEY_SEPARATOR_SCHEMA_V3 =
       new OzoneConfiguration().getObject(DatanodeConfiguration.class)
           .getContainerSchemaV3KeySeparator();
+  private static final int BATCH_SIZE = 10000;
+  private static final int THREAD_COUNT = 10;
```

Comment:
Why not make these optional params as well?

Reply:
Since this change aims to speed up the scan, the thread count and batch size choices can be params.

Reply:
Yes, we can make these parameters configurable, but I didn't do this because I found that after the `THREAD_COUNT`
(number of threads) is greater than 4, the parameters have less of an impact on performance.